### PR TITLE
issue #629 followup: instruction on cargo build and install usage

### PR DESCRIPTION
**Pull request by DaanHoogland**
@shaleh I found only refs to this in the INSTALL file so no changes in readme

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -33,6 +33,11 @@ some of the steps manually.  The more detailed description in the other
 sections of this guide will help you do that, so please refer to those
 sections if you need to.
 
+  0. Before you start: Remacs uses cargo for it's build process. if
```
Comment (shaleh) [inline INSTALL]: `It's` is a conjunction. You want `its`.

File: INSTALL [hunk 2]
```diff
@@ -33,6 +33,11 @@ some of the steps manually.  The more detailed description in the other
...
+     you build as an unpriviledged user and then install as an
+     administrator cargo will rebuild remacs. Make sure you have the
+     right nightly version installed for both users.
+
```
Comment (shaleh) [inline INSTALL]: Maybe mention `rust-toolchain` as the source of that version?

File: INSTALL (hunk 1 above)
Comment (DaanHoogland) [reply INSTALL]: 👍 will tweak

File: INSTALL (hunk 2 above)
Comment (DaanHoogland) [reply INSTALL]: hm, not sure how elaborate to be here,
should i just mention rustc or refer it?

Comment (shaleh) [reply INSTALL]: How about

"Make sure you have the right nightly version as found in the rust-toolchain file." 

Or something like that.

**Pull request merged**